src/components/MyCards/MyCards.jsx

```diff
@@ -13,7 +13,7 @@ export class MyCards extends React.Component {
     render() {
         return (
             <div className={styles.myCards}>
-                <CardList cards={this.props.cards} handleClick={this.props.selectCard} isEnabled={() => this.props.playCard} cy={"my-cards"} />
+                <CardList cards={this.props.cards} handleClick={this.props.selectCard} isEnabled={() => this.props.playCard && !this.props.playedCardId} cy={"my-cards"} />
```

Comment:
Do we want to hide the players' cards once they've picked one?

Reply:
No? That just disables them after they have picked one, right?